sql_extensions/migrations/00001_add_get_top_markets_function.sql

```diff
@@ -0,0 +1,25 @@
+-- migrations/00001_add_get_top_markets_function.sql
+
+-- Function to get the top markets by market_cap
+CREATE OR REPLACE FUNCTION get_top_markets(
+    module_address text
+)
+RETURNS TABLE (
+    transaction_version bigint,
+    data jsonb
+) AS $$
+BEGIN
+    RETURN QUERY
+    WITH latest_entries AS (
+        SELECT DISTINCT ON (e.data->'market_metadata'->'market_id') e.*
+        FROM public.inbox_events e
+        WHERE e.type = module_address || '::emojicoin_dot_fun::State'
```

Comment:
Is there an easy way to use environment variables for these queries @CRBl69? The module address shouldn't change frequently so it'd make more sense and we could avoid needless string concatenation in the query (and remove the input parameter)

Reply:
Here, I would use `WHERE e.type LIKE '%::emojicoin_dot_fun::State'`.